libpod/oci_linux.go

```diff
@@ -187,6 +188,12 @@ func (r *OCIRuntime) conmonPackage() string {
 	return dpkgVersion(r.conmonPath)
 }
 
+type ociError struct {
```

Comment:
This is thoroughly nonstandard and runc-specific, I imagine?

Can we get a comment of some sort to that effect?